packages/db/src/models/base.ts

```diff
@@ -83,6 +83,28 @@ class BasePgModel<ItemType, RecordType extends BaseRecord> {
     return record;
   }
 
+  async getMaxCumulusId(
+    knexOrTransaction: Knex | Knex.Transaction
+  ): Promise<number> {
+    const result = await knexOrTransaction(this.tableName).max('cumulus_id').first();
+    if (!result) {
+      throw new Error('Invalid index query on getMaxCumulusId, max index cannot be returned');
```

Comment:
I'm not sure that I follow what this error means. What does "index query" mean in this context? Would `MAX cumulus_id cannot be returned` be clearer?

Reply:
Functionally it's a catchall as knex's return on .max *can* be undefined, however for all of our defined model classes this shouldn't be possible. 

How about: 41afd4cf30